src/views/LoginView.vue

```diff
@@ -108,7 +108,8 @@
               </div>
 
               <v-btn
-                class="w-100 my-10 text-h6 h-auto py-2"
+                class="w-100 my-10  h-auto py-2"
+                :class="$vuetify.display.xs ? 'text-subtitle-2' : 'text-h6'"
```

Comment:
So per @Haviles04 's suggestion we'd have
```suggestion
                class="w-100 my-10  h-auto py-2 text-subtitle-2 text-md-h6"
```
Which we'd then likely need to lint for line length. @haaarsh619 I think this is nicer